Documentation/development-flow.md

```diff
@@ -13,6 +13,7 @@ don't hesitate to reach out to us on our [Slack](https://Rook-io.slack.com) dev
 1. [GO 1.11](https://golang.org/dl/) or greater installed
 2. Git client installed
 3. Github account
+4. Python 3
```

Comment:
This is too high level of a place to mention the dependency. For example if someone submits a doc PR, they wouldn't need python.

Reply:
Mh. Python 3 is sort of ubiquitous available nowadays. (CentOS 7 being the notable exception here). Is there really a need for making this more specific?